crates/sp-domains/src/lib.rs

```diff
@@ -420,6 +419,59 @@ pub type OpaqueBundles<Block, DomainHash> =
 pub type SignedOpaqueBundles<Block, DomainHash> =
     Vec<SignedOpaqueBundle<NumberFor<Block>, <Block as BlockT>::Hash, DomainHash>>;
 
+pub fn create_dummy_bundle_with_receipts_generic<BlockNumber, Hash, DomainHash>(
```

Comment:
Is it necessary to be defined in `sp-domains`? Why not move to somewhere only accessible and closer to the test/benchmarking space?

Reply:
As working on the benchmark of `pallet-domain-registry` I found `create_dummy_bundle_with_receipts_generic`, `dummy_invalid_state_transition_proof`, and `create_dummy_receipt` are also helpful in `pallet-domain-registry`, thus we may still need to move them to `sp-domains`.

Reply:
I'm not against it, but please make them feature gated in that case.

Reply:
The changes have been pushed to the `Apply review suggestions` commit.